### PR TITLE
12 get api articles id comment count

### PR DESCRIPTION
**Pull request by VenturingForth**
TICKET #12: GET /api/articles/:article_id (comment_count)

TESTING
200: Should retrieve an array of all article objects with the necessary properties (NOW INCLUDING COMMENT COUNT!)"

DOCUMENTATION:
Updated Endpoints JSON

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -133,6 +133,15 @@ describe("GET /api", () => {
                     expect(body.msg).toBe("Bad request");
                 })
             })
+            test("200: Should retrieve an array of all article objects with the necessary properties (now including COMMENT COUNT AS IF WE DIDN'T ALREADY WRITE FOR IT!)", () => {
```
Comment (pippaamy) [inline __tests__/app.test.js:136]: ??? description?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -133,6 +133,15 @@ describe("GET /api", () => {
...
+                return request(app)
+                .get('/api/articles/1')
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.article).toHaveProperty("comment_count", expect.any(Number));
+                    expect(body.article.comment_count).toBe(11);
```
Comment (pippaamy) [inline __tests__/app.test.js:142]: I would include the rest of the properties in this too

**Review by pippaamy — COMMENTED** [review]
Great job :)

File: __tests__/app.test.js (hunk 1 above)
Comment (VenturingForth) [reply __tests__/app.test.js:136]: I was being ridiculous. It's been amended.

File: __tests__/app.test.js (hunk 2 above)
Comment (VenturingForth) [reply __tests__/app.test.js:142]: This has been refactored to test ALL response properties of a get request to the articles endpoint.

**Pull request merged**